configuration/configuration_test.go

```diff
@@ -102,6 +107,36 @@ func TestBlobCorrectPayload(t *testing.T) {
 	testBlob(t, &payload, []byte(fullValidConf), true)
 }
 
+//equalNetSlice creates copies of slices received
+// sort them and then check elements are equal
+func equalNetSlice(slice1, slice2 []string) bool {
```

Comment:
@sameo so I sorted those slices to be able to compare slices element by element, will take a look at this approach. For other matters, I don't know if slices should be sorted or not, that's why I create copies of them and work with those copies instead of modifying the ones read from the configuration.

Other issue I have is having this `func equalNetSlice` function duplicated, any hint on where I could drop this in order to avoid code duplication?